tests/staticfiles_tests/storage.py

```diff
@@ -63,3 +63,17 @@ class SimpleCachedStaticFilesStorage(CachedStaticFilesStorage):
 
     def file_hash(self, name, content=None):
         return 'deploy12345'
+
+
+class MultiCachedStaticFilesStorage(CachedStaticFilesStorage):
+    """
+    A storage class to test pattern substitutions with more than one pattern
+    entry.
```

Comment:
Whatever you think. As for me, I'm more concerned with understanding Django's test suite than using it to provide production-ready code samples. ;-)